packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs

```diff
@@ -0,0 +1,89 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  del,
+  get,
+  getWhereSchemaFor,
+  param,
+  patch,
+  post,
+  requestBody,
+} from '@loopback/rest';
+import { <%= sourceModelClassName %>, <%= targetModelClassName %> } from '../models';
+import { <%= sourceRepositoryClassName %> } from '../repositories';
+
+export class <%= controllerClassName %> {
+  constructor(
+    @repository(<%= sourceRepositoryClassName %>) protected <%= paramSourceRepository %>: <%= sourceRepositoryClassName %>,
+  ) { }
+
+  @get('/<%= sourceModelName %>s/{id}/<%= targetModelName %>s', {
```

Comment:
Please extract the REST endpoints (the model plural names) into shared variables.

```suggestion
  @get('/<%= sourceModelPath %>/{id}/<%= targetModelPath %>', {
```

These variables should be used in all relevant places in all new templates.

Reply:
Resolved.